src/darwin/Framework/CHIP/MTRDeviceController_Internal.h

```diff
@@ -256,6 +256,8 @@ NS_ASSUME_NONNULL_BEGIN
 - (MTRDevice *)deviceForNodeID:(NSNumber *)nodeID;
 - (void)removeDevice:(MTRDevice *)device;
 
+- (NSNumber * _Nullable)syncGetCompressedFabricID;
```

Comment:
If we need this from both on and off the Matter work queue maybe we should cache it after controller startup. Having two separate properties and having to know which one to use depending on where code is running is not ideal.

Reply:
I saw all other usage of the compressed fabric ID eventually queries `_cppCommissioner` for this. And so I kept the pattern.

@bzbarsky-apple Is there any issue with caching compressedFabricID?